lib/travis/build/script.rb

```diff
@@ -240,6 +246,37 @@ def debug_enabled?
         def app_host
           @app_host ||= Travis::Build.config.app_host.to_s.strip.untaint
         end
+
+        def error_message_ary(exception)
+          if exception.is_a? Travis::Build::CompilationError
+            msg = [
+              exception.message
+            ]
+            doc_path = exception.doc_path
+          else
+            msg = [
+              "Unfortunately, we do not know much about this error.",
+              "",
+              "#{exception.class}: #{exception.message}"
```

Comment:
Here, `exception` is a builtin Ruby exception. I do not believe it will contain user input here.